sys-apps/ndctl/ndctl-64.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 autotools linux-info systemd
+
+DESCRIPTION="Utility library for managing the libnvdimm sub-system in the Linux kernel"
+
+HOMEPAGE="http://pmem.io/"
+SRC_URI="https://github.com/pmem/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RESTRICT="test"
+
+RDEPEND="sys-apps/kmod
+	virtual/udev
+	sys-apps/util-linux
+	dev-libs/json-c
+	sys-apps/keyutils"
+
+DEPEND="${RDEPEND}
+	app-text/asciidoc
+	app-text/xmlto"
+
+CONFIG_CHECK="~BLK_DEV_RAM_DAX ~FS_DAX ~X86_PMEM_LEGACY ~LIBNVDIMM ~BLK_DEV_PMEM \
+~ARCH_HAS_PMEM_API ~TRANSPARENT_HUGEPAGE ~MEMORY_HOTPLUG ~MEMORY_HOTREMOVE \
+~ZONE_DEVICE ~FS_DAX_PMD"
+
+src_prepare() {
+	"${S}"/git-version-gen
```

Comment:
Does this need git?

Reply:
Release versions do not need git if it an official release. The script just creates the version.m4 file . However, upstream did forgot to include the "version" file in the tarball so I added a line to generate it in the ebuild. I removed git and compile the ebuild and it worked without it.

Reply:
Wouldn't it be easier to just create that version.m4 from $PV instead?